datafusion/physical-optimizer/src/limit_pushdown.rs

```diff
@@ -256,21 +265,24 @@ pub(crate) fn pushdown_limits(
     pushdown_plan: Arc<dyn ExecutionPlan>,
     global_state: GlobalRequirements,
 ) -> Result<Arc<dyn ExecutionPlan>> {
+    // Call pushdown_limit_helper.
+    // This will either extract the limit node (returning the child), or apply the limit pushdown.
```

Comment:
this might be a good comment to add to the `pushdown_limit_helper` function as well